### PR TITLE
Remove services/projects/goals tab in group page

### LINKED ISSUE #155: Remove services/projects/goals tab in group page
also remove services and products from header

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Bug Fix

## Related issues

Resolve #155

### CODE REVIEW

File: src/components/groups/TeamProfilePreview.tsx [hunk 1]
```diff
@@ -37,14 +37,16 @@ const TeamProfilePreview = ({ groupId }: UserProps): JSX.Element => {
...
                     <PreviewContent>
-                        <NarrowSection title={tr('Quick summary')}>
-                            {nullable(group.supervisor, (supervisor) => (
-                                <StyledSupervisorText size="m" color={gray9}>
-                                    {tr('Supervisor')}
-                                    <UserListItem user={supervisor} />
-                                </StyledSupervisorText>
-                            ))}
-                        </NarrowSection>
+                        {group.supervisor && (
```
Comment (tsumo) [inline src/components/groups/TeamProfilePreview.tsx:40]: Should use nullable

File: src/components/groups/TeamProfilePreview.tsx [hunk 2]
```diff
@@ -37,14 +37,16 @@ const TeamProfilePreview = ({ groupId }: UserProps): JSX.Element => {
...
+                            <NarrowSection title={tr('Quick summary')}>
+                                {nullable(group.supervisor, (supervisor) => (
```
Comment (tsumo) [inline src/components/groups/TeamProfilePreview.tsx:42]: Unnecessary check

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/groups/TeamProfilePreview.tsx (hunk 1 above)
Comment (tsumo) [reply src/components/groups/TeamProfilePreview.tsx]: This fix disappeared from the PR :)

File: src/components/header/PageHeader.tsx [hunk 3]
```diff
@@ -36,15 +29,18 @@ export const PageHeader: React.FC = () => {
                     <PageHeaderLogo />
                 </HeaderLogo>
             }
-            menu={<UserMenu onClick={() => {}} email={session.data?.user.email} />}
+            menu={
+                <HeaderMenu>
+                    <UserMenu onClick={() => {}} email={session.data?.user.email} />
```
Comment (9teen90nine) [inline src/components/header/PageHeader.tsx:34]: Should be either a TODO or a real action

**Review by 9teen90nine — APPROVED** [review]

**Pull request merged**